src/app/pages/applications/catalog/catalog.component.html

```diff
@@ -18,7 +18,7 @@
           </div>
           <div class="badge-area" [class.badge-official]="item.catalog.label === officialCatalog">
             <div class="capitalized-text">
-              {{ item.catalog.label }}
+              {{ item.catalog.label | translate }}
```

Comment:
Here and for `train` below data seems to be coming from middleware => nothing for translator to extract => translate pipe won't do anything.